src/rules/no-stale-state-across-await.ts

```
@@ -0,0 +1,182 @@
+              walkNode(value, skipNestedFunctions);
+            }
+          }
+        }
+
+        // Walk the function body
+        if (node.body) {
+          walkNode(node.body);
+        }
+
+        // Check for violations
+        const setterCallsByName = new Map<string, { name: string; position: number }[]>();
```

ESLint

[error] Replace `string,·{·name:·string;·position:·number·}[]` with `⏎··········string,⏎··········{·name:·string;·position:·number·}[]⏎········`

(prettier/prettier)